webui/src/index.html

```diff
@@ -4,6 +4,17 @@
     <meta charset="UTF-8">
     <link rel="shortcut icon" type="image/png" href="assets/favicon.png">
     <title>Okeears - Objectives and Key Results web application</title>
+
+    <script type="text/javascript">
+        var appInsights=window.appInsights||function(a){
+          function b(a){c[a]=function(){var b=arguments;c.queue.push(function(){c[a].apply(c,b)})}}var c={config:a},d=document,e=window;setTimeout(function(){var b=d.createElement("script");b.src=a.url||"https://az416426.vo.msecnd.net/scripts/a/ai.0.js",d.getElementsByTagName("script")[0].parentNode.appendChild(b)});try{c.cookie=d.cookie}catch(a){}c.queue=[];for(var f=["Event","Exception","Metric","PageView","Trace","Dependency"];f.length;)b("track"+f.pop());if(b("setAuthenticatedUserContext"),b("clearAuthenticatedUserContext"),b("startTrackEvent"),b("stopTrackEvent"),b("startTrackPage"),b("stopTrackPage"),b("flush"),!a.disableExceptionTracking){f="onerror",b("_"+f);var g=e[f];e[f]=function(a,b,d,e,h){var i=g&&g(a,b,d,e,h);return!0!==i&&c["_"+f](a,b,d,e,h),i}}return c
```

Comment:
May be it will be better to do multiline function?

Reply:
This script is generated by MS, we should not touch it (and even read it :). So I would leave it as is. More details (and the script too) here https://docs.microsoft.com/en-us/azure/application-insights/app-insights-javascript